releasenotes/notes/deprecate-span-meta-ad1c96ba55acaac9.yaml

```
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    :py:attr:ddtrace.Span.meta has been deprecated. Use :py:meth:ddtrace.Span.get_tag and :py:meth:ddtrace.Span.set_tag instead.
```

Sorry missed the backticks:

```suggestion
    :py:attr:`ddtrace.Span.meta` has been deprecated. Use :py:meth:`ddtrace.Span.get_tag` and :py:meth:`ddtrace.Span.set_tag` instead.
```